Assets/Scripts/MainMenuController.cs

```diff
@@ -153,4 +139,54 @@ public class MainMenuController : ParentController {
 		}
 	}
 
+	// Handles when a match is found and the new game needs to be added to the user's list
+	private void HandleQguResponse(Dictionary<string, object> response){
+		GameData.SetMatchData(response);
+		GameData.SetMatchQueueData(response);
+
+		if(response["Games"].ToString() == "[]"){
+			Debug.Log("ERROR: QGU did not return any games.");
+
+			return;
+		}
+
+		DisplayRankedGameButton();
+
+		Transform activeGames = GameObject.Find("ActiveCustomGamesContent").transform;
+
+		List<object> matches = Json.ToList(Parse.String(response["Games"]));
+		Dictionary<string, object> matchDataAsDict;
+		int gameId;
+
+		// Populate the list of games with the new game(s)
+		foreach(object match in matches) {
+			matchDataAsDict = Json.ToDict(Parse.String(match));
+			gameId = Parse.Int(matchDataAsDict["ID"]);
+
+			KeyValuePair<int, MatchData> pair = new KeyValuePair<int, MatchData>(gameId, GameData.Matches[gameId]);
+			LoadCustomGame(activeGames, pair, GameData.Matches.Count - 1);
+		}
+
+		// Adjust the size of the scrollable area
+		activeGames.GetComponent<RectTransform>().sizeDelta = new Vector2(Screen.width,
+			300 * (float)GameData.Matches.Count);
+	}
+
+	// Handles when the player wants to cancel searching for a match
+	private void HandleCsResponse(Dictionary<string, object> response){
+		LoadingCircle.Hide();
+
+		// Hide cancel button
+		RectTransform rt = _cancelRankedButton.GetComponent<RectTransform>();
+		rt.anchoredPosition = new Vector3(rt.anchoredPosition.x, rt.anchoredPosition.y + MOVE_BUTTON_DISTANCE);
+
+		// Show set team button
+		rt = _startRankedGame.GetComponent<RectTransform>();
+		rt.anchoredPosition = new Vector3(rt.anchoredPosition.x, rt.anchoredPosition.y - MOVE_BUTTON_DISTANCE);
+	}
+
+	private void HandleSuiResponse(Dictionary<string, object> response){
+		// Do nothing?!
+	}
```

Comment:
I guess if the SUI call returns a success of false, we may want the front end to simply try again but I think this is fine for now.  Since this command is only used for registering the device for notifications, we don't really want to display anything error related in the UI for this command.  Once this command is used for saving changed user preferences/settings, then we likely want to display something in the UI if an error is returned.